bench/wtperf/config.c

```diff
@@ -442,7 +442,11 @@ config_opt(CONFIG *cfg, WT_CONFIG_ITEM *k, WT_CONFIG_ITEM *v)
 		}
 		strp = (char **)valueloc;
 		free(*strp);
-		newstr = dstrdup(v->str);
+		/*
+		 * We duplicate the string to len rather than len+1 as we want
+		 * to truncate the trailing quotation mark.
+		 */
+		newstr = strndup(v->str,  v->len);
```

Comment:
The use of ``strndup`` should follow the same pattern as ``strdup`` - i.e: create a wrapper function that handles errors.